app/views/declaration/declaration_choice.scala.html

```diff
@@ -39,32 +44,65 @@
     }
 }
 
-@main_template(
-    title = Title("declaration.type.description")
-) {
-
-    @components.back_link(backCall, mode)
+@govukLayout(
+    title = Title("declaration.type.description"),
+    backButton = Some(BackButton(messages("site.backToSelectionPage"), backCall))){
 
-    @helper.form(controllers.declaration.routes.DeclarationChoiceController.submitChoice(mode), 'autoComplete -> "off") {
-        @helper.CSRF.formField
+    @formHelper(action = controllers.declaration.routes.DeclarationChoiceController.submitChoice(), 'autoComplete -> "off") {
```

Comment:
Is mode not missing here ?

Reply:
Good spot.  Unfortunately, it looks like no test picked that up